packages/js/src/Modules/Verto/index.ts

```diff
@@ -20,7 +20,69 @@ export default class Verto extends BrowserSession {
     return isValidOptions(this.options);
   }
 
+  /**
+   * Makes a new outbound call.
+   *
+   * This method receives an object `options` with the following properties:
```

Comment:
I don't see any differentiation between required and optional options, could you specify?

Reply:
To clarify: did you manually test the required and optional `CallOptions` as a part of this PR? For example, I'm pretty sure you can successfully place a call even if `remoteCallerName` and `remoteCallerNumber` are omitted--in fact, this isn't documented below at all. However, those two fields are marked as required in the `CallOptions` interface. Could you update the interface if those options are in fact not required to place a call?

Reply:
I tested some required and some optional params, not all.

The only required param to the method `newCall` is the `destinationNumber` the others are optional

Reply:
Done!